applications/openshift/controller/controller_use_service_account/oval/shared.xml

```diff
@@ -0,0 +1,29 @@
+<def-group>
+  <definition class="compliance" id="controller_use_service_account" version="1">
+    <metadata>
+      <title>Ensure that use-service-account-credentials is true</title>
+      <affected family="unix">
+        <platform>multi_platform_ocp</platform>
+      </affected>
+      <description>Ensure use-service-account-credentials is true.</description>
+    </metadata>
+    <criteria operator="AND">
+      <criterion comment="use-service-account-credentials is configured" test_ref="test_controller_use_service_account" negate="true" />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="use-service-account-credentials is configured" id="test_controller_use_service_account" version="1">
+    <ind:object object_ref="object_controller_use_service_account" />
+    <ind:state state_ref="state_controller_use_service_account" />
+  </ind:textfilecontent54_test>
+
+  <ind:textfilecontent54_object id="object_controller_use_service_account" version="1">
+    <ind:filepath>/etc/origin/node/node-config.yaml</ind:filepath>
+    <ind:pattern operation="pattern match">^[\s]*kubernetesMasterConfig\:(?:[^\n]*\n+)+?[\s]*controllerArguments\:[\s]*[\n]+[\s]*-[\s]+(\S+).*$</ind:pattern>
+    <ind:instance datatype="int">1</ind:instance>
+  </ind:textfilecontent54_object>
+
+  <ind:textfilecontent54_state id="state_controller_use_service_account" version="1">
+    <ind:subexpression datatype="string" operation="pattern match">^.*use-service-account-credentials=false.*$</ind:subexpression>
```

Comment:
This is wrong. Should be
`<ind:pattern operation="pattern match">^[\s]*kubernetesMasterConfig\:(?:[^\n]*\n+)+?[\s]*controllerArguments\:[\s]*[\n]+[\s]*use-service-account-credentials\:[\s]*[\n]+[\s]*-[\s]+(\S+).*$</ind:pattern>`
and
`<ind:subexpression datatype="string" operation="pattern match">^'false'$</ind:subexpression>

Reply:
Hmm, ok. That also means the kubelet OVAL where I stole this from needs updating ;)

Reply:
fixed in d69b679. thanks @redhatrises